app/pyproject.toml

```diff
@@ -53,7 +53,7 @@ isort = "5.6.4"
 google-cloud-storage = "1.32.0"
 django-storages = "1.10.1"
 graphene-django = "^2.13.0"
-mozilla-nimbus-shared = "^1.0.0"
+mozilla-nimbus-shared = "^1.1.0"
```

Comment:
Looks like experimenter/app/experimenter/legacy-ui/rapid/package.json is still on v0.0.9. Not sure if that creates any potential conflicts.

https://github.com/mozilla/experimenter/blob/6e66462cbf57a721018b8f86251ad8a9ee0b1a7e/app/experimenter/legacy-ui/rapid/package.json#L7

Reply:
No it needs that version, so that's correct.

Reply:
But thnx for checking 👍